applications/GeoMechanicsApplication/custom_constitutive/mohr_coulomb_with_tension_cutoff.cpp

```diff
@@ -317,4 +317,25 @@ void MohrCoulombWithTensionCutOff::FinalizeMaterialResponseCauchy(ConstitutiveLa
     mStressVectorFinalized = mStressVector;
 }
 
+void MohrCoulombWithTensionCutOff::save(Serializer& rSerializer) const
+{
+    rSerializer.save("ConstitutiveLawDimension", mpConstitutiveDimension);
+    rSerializer.save("StressVector", mStressVector);
+    rSerializer.save("StressVectorFinalized", mStressVectorFinalized);
+    rSerializer.save("StrainVectorFinalized", mStrainVectorFinalized);
+    rSerializer.save("CoulombYieldSurface", mCoulombYieldSurface);
+    rSerializer.save("TensionCutOff", mTensionCutOff);
+    rSerializer.save("mIsModelInitialized", mIsModelInitialized);
+}
+
+void MohrCoulombWithTensionCutOff::load(Serializer& rSerializer)
+{
+    rSerializer.load("ConstitutiveLawDimension", mpConstitutiveDimension);
+    rSerializer.load("StressVector", mStressVector);
+    rSerializer.load("StressVectorFinalized", mStressVectorFinalized);
+    rSerializer.load("StrainVectorFinalized", mStrainVectorFinalized);
+    rSerializer.load("CoulombYieldSurface", mCoulombYieldSurface);
+    rSerializer.load("TensionCutOff", mTensionCutOff);
+    rSerializer.save("mIsModelInitialized", mIsModelInitialized);
```

Comment:
See previous comment
```suggestion
    rSerializer.save("IsModelInitialized", mIsModelInitialized);
```

Reply:
fixed